backend/src/main/java/com/woowacourse/momo/controller/auth/AuthArgumentResolver.java

```diff
@@ -0,0 +1,48 @@
+package com.woowacourse.momo.controller.auth;
+
+import com.woowacourse.momo.exception.MomoException;
+import com.woowacourse.momo.exception.code.AuthErrorCode;
+import com.woowacourse.momo.service.auth.JwtManager;
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import org.springframework.core.MethodParameter;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.support.WebDataBinderFactory;
+import org.springframework.web.context.request.NativeWebRequest;
+import org.springframework.web.method.support.HandlerMethodArgumentResolver;
+import org.springframework.web.method.support.ModelAndViewContainer;
+
+@Component
+@RequiredArgsConstructor
+public class AuthArgumentResolver implements HandlerMethodArgumentResolver {
+
+    private static final String AUTHORIZATION = "Authorization";
+    private static final String BEARER = "Bearer ";
+
+    private final JwtManager jwtManager;
+
+    @Override
+    public boolean supportsParameter(MethodParameter parameter) {
+        return parameter.hasParameterAnnotation(AuthAttendee.class) &&
+                parameter.getParameterType().equals(long.class);
```

Comment:
비교 연산자도 함께 줄바꿈하는게 코드 읽는 방향을 고려해서 가독성이 좋을 것 같아요!
```suggestion
    @Override
    public boolean supportsParameter(MethodParameter parameter) {
        return parameter.hasParameterAnnotation(AuthAttendee.class) 
               && parameter.getParameterType().equals(long.class);
    }
```

Reply:
좋습니다

Reply:
좋아요~ 👍